Dockerfile

```diff
@@ -99,9 +99,9 @@ RUN if [ "x$skip_dev_deps" = "x" ] ; then pip install -r requirements_dev.txt ;
 COPY requirements.txt ./
 RUN pip install -r requirements.txt
 
-COPY --chown=redash . /app
-COPY --from=frontend-builder --chown=redash /frontend/client/dist /app/client/dist
-RUN chown redash /app
+COPY . /app
+COPY --from=frontend-builder /frontend/client/dist /app/client/dist
+RUN chown -R redash /app
```

Comment:
This was probably better before - I wonder if it changed since the version pjohnson based his one on?